### PR TITLE
Add Stateful features to collection

### PR DESCRIPTION
**Pull request by christian-bromann**
This PR adds devcontainer features to the collection.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -488,3 +488,8 @@
   contact: https://github.com/earthly/devcontainer-features/issues
   repository: https://github.com/earthly/devcontainer-features
   ociRefernce: ghcr.io/earthly/devcontainer-features
+- name: Runme Container Features
+  maintainer: Stateful, Inc.
+  contact: https://github.com/stateful/devcontainer-features/issues
+  repository: https://github.com/stateful/devcontainer-features
+  ociRefernce: https://ghcr.io/stateful/devcontainer-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/stateful/devcontainer-features
```

Looks like a typo with `ociReference`, and the ghcr ref has an unwanted `https` prefix. @christian-bromann can you help update? Thanks! 

image

File: _data/collection-index.yml [hunk 2]
```diff
@@ -488,3 +488,8 @@
   contact: https://github.com/earthly/devcontainer-features/issues
   repository: https://github.com/earthly/devcontainer-features
   ociRefernce: ghcr.io/earthly/devcontainer-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/earthly/devcontainer-features
```

Looks like the previous entry has a typo with  `ociReference`. @christian-bromann if possible, can you help fix that as well?

**Comment by christian-bromann** [comment]
@samruddhikhandale thanks for the review, good catch! Good old copy&pasting error 🙈

**Review by samruddhikhandale — APPROVED** [review]
Nice, looks good. Thanks! ⚡

**Pull request merged**